script/DeployENSPayload.s.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/console.sol';
+import 'forge-std/Script.sol';
+import {ENSListingPayload} from 'src/contracts/ENSListingPayload.sol';
+
+contract DeployENSListingPayload is Script {
+    function run() public {
+        vm.startBroadcast();
+
+        address ensListingPayload = address(new ENSListingPayload());
+        console.log('ENS Listing Payload address', ensListingPayload);
```

Comment:
We commented that not necessary, as Foundry will log by default. So no declaration of `ensListingPayload` needed